src/checkout-buttons/strategies/checkout-button-method.ts

```diff
@@ -0,0 +1,4 @@
+export enum CheckoutButtonMethod {
```

Comment:
Can you name this enum as `CheckoutButtonMethodType`? So it's clearer that it is not an object?